library/Opus/EnrichmentKey.php

```diff
@@ -186,4 +197,81 @@ public function getEnrichmentType()
         $typeObj->setOptions($this->getOptions());
         return $typeObj;
     }
+
+    /**
+     * Ändert den Namen des vorliegenden EnrichmentKey in allen Dokumenten, die
+     * Enrichments mit diesem Namen verwenden.
+     *
+     * Achtung: diese Methode ändert *nicht* den Namen des EnrichmentKeys in der
+     * Tabelle enrichmentkeys.
+     *
+     * @param string $newName neuer Name des EnrichmentKey
+     * @param string | null $oldName ursprünglicher Name des EnrichmentKey, wenn null, dann
+     *                      wird der aktuelle Name des EnrichmentKey verwendet
+     */
+    public function rename($newName, $oldName = null)
```

Comment:
Sollte die "nur in Dokumenten" Funktion im Namen reflektiert werden, z.B. "renameInDocuments"? Es sollte trotzdem eine *rename*-Funktion geben. Ich denke da an die *delete*-Funktion von `Opus_Document`, die eine Status-Änderung durchführt, als Beispiel. Rename würde also den Namen ändern in den Dokumenten und in EnrichmentKeys, wenn dort vorhanden.